src/components/example/certificate/CertificateForm.tsx

```diff
@@ -97,17 +112,68 @@ const CertificateForm: FC<{ initialValues: ICertificate }> = ({
   const handleClose = () => {
     setIsDialogOpen(false);
   };
+  const closeUserLookup = () => {
+    setIsUserLookup(false);
+  };
   const { translate } = useTranslate();
+  interface Column {
```

Comment:
Please move both interface Column and columns definition outside of component. In general, it is bad practice to keep code component doesn't absolutely need inside of component. This makes code harder to read and component unnecessary complex. Additionally, Column interface is something you could use in all tables, not just this one and columns is constant, it doesn't change so define it as constant.